9_collaborator_data.yml

```diff
@@ -0,0 +1,120 @@
+target_default: 9_collaborator_data
+
+packages:
+  - sf
+  - tidyverse
+  - lwgeom
+  - smoothr
+  - geojsonio
+  - geojsonsf
+  - ggplot2
+
+sources:
+  - 1_network/src/calc_distance_functions.R
+  - 1_network/src/geo_fabric_functions.R
+  - 2_observations/src/data_munge_functions.R
+  - 9_collaborator_data/src/makerspace_geojson.R
+  - 9_collaborator_data/src/write_functions.R
+
+targets:
+  9_collaborator_data:
```

Comment:
I can build this target without any computations :+1:

I'm out of time to look in detail at the functions or outputs in this .yml, but I like the overall framework (.yml for all collaborator outputs) and assume these targets will get extra vetting by us & collaborators in the future.